pkg/virt-controller/services/renderresources.go

```diff
@@ -212,14 +212,20 @@ func WithCPUPinning(cpu *v1.CPU) ResourceRendererOption {
 			}
 		}
 
-		// allocate 1 more pcpu if IsolateEmulatorThread request
+		// allocate pcpus for emulatorThread if IsolateEmulatorThread is requested
 		if cpu.IsolateEmulatorThread {
-			emulatorThreadCPU := resource.NewQuantity(1, resource.BinarySI)
+			const emulatorThreadCPU = 1
+			emulatorThreadCPUs := resource.NewQuantity(emulatorThreadCPU, resource.BinarySI)
 			limits := renderer.calculatedLimits[k8sv1.ResourceCPU]
-			limits.Add(*emulatorThreadCPU)
+			if v1.CPUManagerPolicyBetaOptions(cpuManagerPolicyBetaOptionsAnnotationValue) == v1.FullpCPUsOnly {
```

Comment:
In case the function's input param is changed to `v1.CPUManagerPolicyBetaOptions`:
```suggestion
if cpuManagerPolicyBetaOption == v1.FullpCPUsOnly {
```

Reply:
pending your answer on the related comment.

Reply:
DONE